src/instructlab/eval/mt_bench_common.py

```diff
@@ -382,3 +382,9 @@ def check_data(questions, model_answers, ref_answers, models, judges):
 def get_model_list(answer_file):
     logger.debug(locals())
     return [os.path.splitext(os.path.basename(answer_file))[0]]
+
+
+def set_api_key(api_key):
```

Comment:
Suggestion: Make this be get_openai_client and include the line just below from the two callers as well.

Reply:
updated